index.html

```diff
@@ -305,6 +305,109 @@ <h2>Disputes</h2>
+be targeted to a specific <a>verifier</a> by using a Linked Data Proof
+that includes a <code>domain</code> and <code>challenge</code>. This also
+helps prevent a <a>verifier</a> from reusing a <a>verifiable presentation</a>
+as their own.
+      </p>
+
+      <p>
+The following <a>verifiable presentation</a> is for authenticating to
+a website, <code>https://example.com</code>. Note that the <code>domain</code>
+value can be any string or URI, and the <code>challenge</code> should be a
+randomly generated string.
+      </p>
+
+
+      <pre class="example nohighlight" title="A targeted verifiable presentation">
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1"
+  ],
+  "type": "VerifiablePresentation,
+  "verifiableCredential": { ... },
+  "proof": {
+    "type": "Ed25519Signature2018",
+    "created": "2019-08-13T15:09:00Z",
+    <span class="highlight">"challenge": "d1b23d3...3d23d32d2",
+    "domain": "https://example.com",</span>
+    "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..uyW7Hv
+      VOZ8QCpLJ63wHode0OdgWjsHfJ0O8d8Kfs55dMVEg3C1Z0bYUGV49s8IlTbi3eXsNvM63n
+      vah79E-lAg",
+    "proofPurpose": "authentication"
+  }
+}
+      </pre>
+
+    </section>
+
+    <section>
+      <h4>Using the JWT aud claim</h3>
+
+      <p>
+The JWT <code>aud</code> <a>claim</a> name refers to (i.e., identifies) the
+intended audience of the <a>verifiable presentation</a> (i.e., the verifier(s)).
+Consequently this is an alternative to the Linked Data Proof method specified
+above. It lets the <a>holder</a> indicate which <a>verifier(s)</a> it allows
+to verify the <a>verifiable presentation</a>. Any JWT-compliant <a>verifier</a>
+that is not identified in the <code>aud</code> is required to
+reject the JWT (see <a href="https://tools.ietf.org/html/rfc7519">RFC 7519</a>).
+      </p>
+      <p>
+<a href="https://tools.ietf.org/html/rfc7519">RFC 7519</a> defines
+<code>aud</code> as "an array of case-sensitive strings, each containing a
+<code>StringOrURI</code> value". For use in a <a>verifiable presentation</a>,
+we recommend that this be restricted to a single URI value, equal to the URI
+of the intended verifier.
+      </p>
+      <p>
+The <a href="https://w3c.github.io/vc-data-model/">data model specification</a>
+provides no guidance of how to transform this JWT <a>claim</a> into a property
+of the <a>verifiable presentation</a>, nor vice versa. We recommend that the
+<code>aud</code> JWT <a>claim</a> be mapped to the <code>verifier</code> 
+<a>property</a> of the <a>verifiable presentation</a>.
+      </p>
+
+      <pre class="example nohighlight" title="An example of a
+targeted verifiable presentation using the verifier property">
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1"
+  ],
+  "type": "VerifiablePresentation",
+  "verifiableCredential": ["..."],
+  "holder": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+  "verifier": "https://some.verifier.com"
```

Comment:
This is not a property in the context so it isn't valid. This example needs to change in some way (a new context or be removed, etc).

Reply:
I suggest we add the term verifier to the core context. Can you do this please?

Reply:
I have updated the example by adding the new context "https://www.w3.org/2018/credImpGuide/v1"
How do we ask W3C to publish a context for the implementation guide.

Reply:
Manu has suggested we use the new context 
https://www.w3.org/2019/credentials/v1.1
and ask the CCG to publish this. Could Manu and/or the chairs request the CCG to do this please

Reply:
Done